### DISCUSSION TITLE
使用导航栏 TabBar在苹果X显示问题

### DISCUSSION THREAD

**Discussion by xuwenfeng93** [TaleLin/lin-ui#1289] [求助] [answered]
![image](https://user-images.githubusercontent.com/48546665/106244435-d625d080-6245-11eb-8424-a42dfe877bc1.png)
如图显示，导航栏在苹果下面会被小黑线挡住

**Comment by juzi214032** [comment, marked as answer]
0.9.3 版本会修复此问题

**Reply by juzi214032** [reply]
0.9.4 版本已修复
